workspaces/libnpmpublish/lib/publish.js

```diff
@@ -144,19 +144,27 @@ const buildMetadata = async (registry, manifest, tarballData, spec, opts) => {
       digest: { sha512: integrity.sha512[0].hexDigest() },
     }
 
-    // Ensure that we're running in GHA, currently the only supported build environment
-    if (ciInfo.name !== 'GitHub Actions') {
+    if (ciInfo.GITHUB_ACTIONS) {
+      // Ensure that the GHA OIDC token is available
+      if (!process.env.ACTIONS_ID_TOKEN_REQUEST_URL) {
+        throw Object.assign(
+          /* eslint-disable-next-line max-len */
+          new Error('Provenance generation in GitHub Actions requires "write" access to the "id-token" permission'),
+          { code: 'EUSAGE' }
+        )
+      }
+    } else if (ciInfo.GITLAB) {
+      // Ensure that the Sigstore OIDC token is available
+      if (!process.env.SIGSTORE_ID_TOKEN) {
+        throw Object.assign(
+          /* eslint-disable-next-line max-len */
+          new Error('Provenance generation in GitLab CI requires "SIGSTORE_ID_TOKEN" with "sigstore" audience to be present in "id_tokens". See https://docs.gitlab.com/ee/ci/secrets/id_token_authentication.html for more info.'),
```

Comment:
```suggestion
          new Error('Provenance generation in GitLab CI requires "SIGSTORE_ID_TOKEN" with "sigstore" audience to be present in "id_tokens". For more info see:\nhttps://docs.gitlab.com/ee/ci/secrets/id_token_authentication.html'),
```

This helps folks w/ screen readers find, copy, and paste the url in isolation.

Reply:
Done.